UKHomeOffice/sgmr-service
# Version 2.2.15

SPCR-540
### AC / Description of changes

- change `email` to `username` on the forgotten password page
- update wording once forgotten password submitted, regardless of success of fail response from API
- remove help text for `email` on create account page

### To Test
- go to create an account
- > there should be no help text for email

- go to sign in
- click on problems signing in
- > the field label should be username
- enter registered email address
- click submit
- > the confirmation page should say `If you have an active account a password reset link has been sent to your registered email address.`
- > you should receive a password reset email

- go back to forgotten password page
- enter a non-registered email address
- > the confirmation page should say `If you have an active account a password reset link has been sent to your registered email address.`
- click submit
- > you should not receive an email to that email address

- go back to forgotten password page
- enter something that is not an email address
- click submit
- > you should receive an error that is it not a valid email address


### Notes

---

**Remember to merge to feature branches not `master`.**
